### PR TITLE
LPS-174947 Create configuration page to set the max number of pages generated as preview

### PR DESCRIPTION
**Pull request by AliciaGarciaGarcia**
_edit_: Updated after feedback
image

On this PR we are only creating the several configurations (System/Company/Site) that can be shown on the settings.
The messages are the same

I will ask on Slack about the messages to the PM and the designer to check them

Thanks!

### CODE REVIEW

**Comment by liferay-lima** [comment]
ci:test:sf

**Comment by liferay-lima** [comment]
ci:test:relevant

**Comment by AliciaGarciaGarcia** [comment]
ci:stop

**Comment by AliciaGarciaGarcia** [comment]
I will force push some changes as told on slack :)

**Comment by AliciaGarciaGarcia** [comment]
The new messages : 

image

**Comment by adolfopa** [comment]
Shouldn't this configuration be in the same module as the pdf processor? (`document-library-preview-pdf`).

Also, I'd remove the "limit" from the configuration name, there's no need to be so specific.

**Comment by AliciaGarciaGarcia** [comment]
ci:test:sf

File: modules/apps/document-library/document-library-preview-pdf/src/main/java/com/liferay/document/library/preview/pdf/internal/configuration/PDFPreviewSystemConfiguration.java [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+package com.liferay.document.library.preview.pdf.internal.configuration;
+
+import aQute.bnd.annotation.metatype.Meta;
+
+import com.liferay.portal.configuration.metatype.annotations.ExtendedObjectClassDefinition;
+
+/**
+ * @author Alicia García
+ */
+@ExtendedObjectClassDefinition(category = "documents-and-media")
```
[the remark this answers is not in the document]
Comment (AliciaGarciaGarcia) [reply modules/apps/document-library/document-library-preview-pdf/src/main/java/com/liferay/document/library/preview/pdf/internal/configuration/PDFPreviewSystemConfiguration.java:24]: yes, the default value is system
https://github.com/liferay/liferay-portal/blob/c635c74ecbf43f8199ffa7d8fe0af178b22e82f3/modules/apps/static/portal-configuration/portal-configuration-metatype-api/src/main/java/com/liferay/portal/configuration/metatype/annotations/ExtendedObjectClassDefinition.java#L57

**Pull request closed without merging**